koku/masu/processor/tasks.py

```diff
@@ -1266,3 +1274,43 @@ def validate_daily_data(schema, start_date, end_date, provider_uuid, ocp_on_clou
         data_validator.check_data_integrity()
     else:
         LOG.info(log_json(msg="skipping validation, disabled for schema", context=context))
+
+
+@celery_app.task(name="masu.processor.tasks.process_openshift_on_cloud_trino", queue=DownloadQueue.DEFAULT, bind=True)
+def process_openshift_on_cloud_trino(
+    self, reports_to_summarize, provider_type, schema_name, provider_uuid, tracing_id
+):
+    """Process OCP on Cloud data into managed tables for summary"""
+    reports_deduplicated = deduplicate_summary_reports(reports_to_summarize, manifest_list=[])
+    for report in reports_deduplicated:
+        schema_name = report.get("schema_name")
+        ctx = {"provider_type": provider_type, "schema_name": schema_name, "provider_uuid": provider_uuid}
+        if provider_type not in Provider.MANAGED_OPENSHIFT_ON_CLOUD_PROVIDER_LIST:
+            LOG.info(
+                log_json(tracing_id, msg="provider type not valid for COST-5129 OCP on cloud processing", context=ctx)
+            )
+            continue
+        if not is_managed_ocp_cloud_processing_enabled(schema_name):
+            LOG.info(
+                log_json(tracing_id, msg="provider not enabled for COST-5129 OCP on cloud processing", context=ctx)
+            )
+            continue
+        with ReportManifestDBAccessor() as manifest_accesor:
+            tracing_id = report.get("tracing_id", report.get("manifest_uuid", "no-tracing-id"))
+
+            if not manifest_accesor.manifest_ready_for_summary(report.get("manifest_id")):
+                LOG.info(log_json(tracing_id, msg="manifest not ready for summary", context=report))
+                continue
+
+        LOG.info(log_json(tracing_id, msg="report to summarize", context=report))
+
+        months = get_months_in_date_range(report)
+        for month in months:
+            start_date = month[0]
+            end_date = month[1]
+            # invoice_month = month[2]
```

Comment:
I assume this is just a future place holder for when we use this flow for GCP.

Reply:
Yeah, I'm not entirely sure it'll be used in GCP but presumably it will since we return it for GCP in other places so I left the comment here just for the future.